docs/source/what_is_jupyter.md

```diff
@@ -0,0 +1,412 @@
+# What is Jupyter?
+
+"Jupyter" can mean a lot of things, so let's start from the beginning, and
+break it down:
+
+First and foremost, "Project Jupyter" is a large umbrella project that covers
+many different software offerings and tools. That includes "Jupyter Notebook"
+and "JupyterLab", which are both popular notebook-editor programs. **The**
+**Jupyter project, and its subprojects, all center around providing tools**
+**(and standards) for interactive computing with computational notebooks.**
+If you're not familiar with some of those things, you can take a guided tour
+below that explains each one from the start.
+
+The term "Jupyter" is often used as a shorthand to refer to one of those
+products or ideas, which can cause confusion. Let's take a look at each piece,
+and provide some clarity, below.
+
+# What is a "Computational Notebook" anyway?
```

Comment:
```suggestion
# What is a computational notebook?
```

"Computational Notebook" is not a proper noun. I rarely see the term "computational" used — should we just use "notebook" (in lowercase) here?

Reply:
Resolved in another comment.